README.md

```diff
@@ -39,10 +39,29 @@ $ ./gradlew docker
 
 You can run the fat jar as follows.
 
+
+```console
+
+$ java -jar scalar-admin-<version>-all.jar -c <COMMAND> -a <IP1:Port1,IP2:Port2,...>
+```
+
+The command above can take effect on multiple ScalarDB or ScalarDL instances in one shot.
+
+**NOTE** If you previously used the following command to pause and unpause Scalar pods in Kubernetes clusters,
+we recommend switching to [Scalar Admin for Kubernetes](https://github.com/scalar-labs/scalar-admin-for-kubernetes).
+
 ```console
 $ java -jar scalar-admin-<version>-all.jar -c <COMMAND> -s <SRV_SERVICE_URL>
 ```
 
+Using the `-s` option to pause and unpause Scalar pods in a Kubernetes cluster is error-prone.
+Scalar Admin doesn't check for orchestration changes during the pause/unpause period, while Scalar Admin for Kubernetes is tolerant of Kubernetes orchestration changes.
+
+This command is an FYI and we do not recommend using it.
+Please either use the `-a` option to explicitly pause or unpause Scalar instances, or consider using Scalar Admin for Kubernetes to pause and then unpause Scalar pods in a tolerant manner.
```

Comment:
I think these sentences mix up two things:

- `Scalar Admin` is error-prone.
- The `-s` option has a known issue (The SRV record returns an asynchronous result).

So, I feel it's a bit confusing.

I think we should not explain these two things at the same place to avoid unnecessary confusion.

I think we should explain these two things separately as follows:

1. The old (deprecated) `-s` option has a known issue.
   - So, we don't recommend to use the `-s` option.
   - Please use the `-a` option instead of the `-s` option.
2. However, we recommend using `Scalar Admin for Kubernetes` instead of `Scalar Admin` unless you have special reasons.
   - This is because `Scalar Admin` is error-prone in the Kubernetes environment.
   - We recommend using Kubernetes as a platform for Scalar products and Scalar Admin for Kubernetes for backup operations.

What do you think?

Reply:
@kota2and3kan 

I understood what you mean.

Let me try to revise it.